searx/settings.yml

```
@@ -477,14 +477,14 @@ engines:
     shortcut: ask
     disabled: true
 
-  # - name: azure
-  #   engine: azure
-  #   shortcut: az
-  #   categories: [it, cloud]
-  #   azure_tenant_id: "your_tenant_id"
-  #   azure_client_id: "your_client_id"
-  #   azure_client_secret: "your_client_secret"
-  #   disabled: true
+  - name: azure
+    engine: azure
+    shortcut: az
+    categories: [it, cloud]
+    azure_tenant_id: "your_tenant_id"
+    azure_client_id: "your_client_id"
+    azure_client_secret: "your_client_secret"
```

```suggestion
    # azure_tenant_id: "your_tenant_id"
    # azure_client_id: "your_client_id"
    # azure_client_secret: "your_client_secret"
```
This should still be commented out so that when the admin sets `inactive: false` but doesn't provide any of these settings, the engine can detect that and throw a warning that e.g. `azure_tenant_id` is missing.